internal/dataplane/fallback/graph_dependencies_kong.go

```diff
@@ -140,3 +141,36 @@ func resolveTCPIngressDependencies(cache store.CacheStores, tcpIngress *kongv1be
 func resolveKongServiceFacadeDependencies(cache store.CacheStores, kongServiceFacade *incubatorv1alpha1.KongServiceFacade) []client.Object {
 	return resolveDependenciesForServiceLikeObj(cache, kongServiceFacade)
 }
+
+// resolveKongCustomEntityDependencies resolves potential dependencies for a KongCustomEntities object:
+// - KongPlugin
+// - KongClusterPlugin.
+func resolveKongCustomEntityDependencies(cache store.CacheStores, obj *kongv1alpha1.KongCustomEntity) []client.Object {
+	if obj.Spec.ParentRef == nil {
+		return nil
+	}
+
+	parentRef := *obj.Spec.ParentRef
+	groupMatches := parentRef.Group != nil && *parentRef.Group == "configuration.konghq.com"
```

Comment:
Could we replace the group `configuration.konghq.com` here to the constant defined in the API packages?